### DISCUSSION TITLE
Get current logged in username

### DISCUSSION THREAD

**Discussion by micebrain** [Budibase/budibase#3128] [General]
Hi,
I am doing a test with an external postgres database and using the builtin user table in Budibase. Is it possible to get the current logged in username that could be used as a filter and update for the external DB?
I know this is possible when using tables created in Budibase using relationships. But I see that relationship can only be created within the same DB. 
It would be nice to have a way to access to the logged in user either perhaps via a global variable that can be passed to external DB or API.

**Comment by mjashanks** [comment]
Hi There.

This should be possible. Wherever you are in your application, you have access to the `{{ Current User.email }}` binding. So, you should be able to pass this to your SQL db as a filter.

![image](https://user-images.githubusercontent.com/3524181/138681789-19a97b56-32ec-45ce-ba92-483b6b2cfb2d.png)

**Reply by micebrain** [reply]
Thanks!
